subgraphs/erc721-metadata/src/mappings/token.ts

```diff
@@ -32,29 +31,29 @@ export function normalize(strValue: string): string {
 }
 
 export function createToken(
+  event: Transfer,
```

Comment:
It seems like it is not necessary to make this event specifically type `Transfer`. To be more generic you could use `ethereum.Event` for the type so it doesn't have to come from a transfer.

Reply:
makes sense to me! will update that 👍 
also will increment the version in a separate PR (these metadata/holders subgraphs have no versioning mechanism currently)